src/components/dialog/__snapshots__/__spec__.js.snap

```diff
@@ -0,0 +1,439 @@
+      transitionEnter={true}
+      transitionEnterTimeout={500}
+      transitionLeave={true}
+      transitionLeaveTimeout={500}
+      transitionName="modal-background"
+    />
+  </div>
+</Portal>
+`;
+
+exports[`Dialog render when dialog is open has the correct content, tags, elements etc 1`] = `
+<Portal
+  open={true}
+>
+  <div
+    className="carbon-dialog foo"
+    data-component="dialog"
+    data-element="bar"
+    data-role="baz"
+  >
+    <CSSTransitionGroup
+      component="div"
+      transitionAppear={false}
+      transitionEnter={true}
+      transitionEnterTimeout={500}
+      transitionLeave={true}
+      transitionLeaveTimeout={500}
+      transitionName="modal"
+    >
+      <div
+        aria-describedby="carbon-dialog-subtitle"
+        aria-labelledby="carbon-dialog-title"
+        className="carbon-dialog__dialog carbon-dialog__dialog--small carbon-dialog__dialog--has-height"
+        data-component="dialog"
+        data-element="bar"
+        data-role="baz"
+        onBlur={[Function]}
+        role="dialog"
+        style={
+          Object {
+            "minHeight": "500px",
+          }
+        }
+        tabIndex={0}
+      >
+        <div
+          className="carbon-dialog__title carbon-dialog__title--has-subheader"
+        >
+          <Heading
+            divider={true}
+            separator={false}
+            subheader="Test"
+            subtitleId="carbon-dialog-subtitle"
+            title="Test"
+            titleId="carbon-dialog-title"
+          />
+        </div>
+        <div
+          className="carbon-dialog__content"
+        >
+          <div
+            className="carbon-dialog__inner-content"
+            style={
+              Object {
+                "minHeight": "calc(500px - 40px)",
+              }
+            }
+          >
+            <Button
+              as="secondary"
+              disabled={false}
+              size="medium"
+              subtext=""
+              theme="blue"
+            >
+              Button
+            </Button>
+            <Button
+              as="secondary"
+              disabled={false}
+              size="medium"
+              subtext=""
+              theme="blue"
+            >
+              Button
+            </Button>
+          </div>
+        </div>
+        <Icon
+          bgSize="small"
+          className="carbon-dialog__close"
+          data-element="close"
+          onBlur={[Function]}
+          onClick={[Function]}
+          tabIndex="0"
+          type="close"
+        />
+      </div>
+    </CSSTransitionGroup>
+    <CSSTransitionGroup
+      component="div"
+      transitionAppear={false}
+      transitionEnter={true}
+      transitionEnterTimeout={500}
+      transitionLeave={true}
+      transitionLeaveTimeout={500}
+      transitionName="modal-background"
+    >
+      <div
+        className="carbon-modal__background"
+      />
+    </CSSTransitionGroup>
+  </div>
+</Portal>
```

Comment:
this feels as though there are a few too many snapshots, can they be condensed into a fewer? or tests that just examine the props you care about?